src/list/components/item-summary.js

```diff
@@ -74,6 +74,7 @@ export default function ItemSummary({className, id, title, username, verbose,
                    $length={trimmedUsername.length}>
           <div data-name="subtitle" className={styles.subtitle}>no uSERNAMe</div>
         </Localized>
+        <div className={styles.info}></div>
```

Comment:
This inadvertently causes a breakage in `test/unit/list/components/item-summary-test.js` because more than one element matches `div > div`.

I think the most expedient way to fix htis is to change the [`merge classNames` test](https://github.com/mozilla-lockbox/lockbox-addon/blob/c36df2c25d65161c72c3eb5021a153879040d876/test/unit/list/components/item-summary-test.js#L47) to explicitly set an item id and then use it as `data-item-id` when `find()`ing the item.